src/modules/folder/folder.usecase.ts

```diff
@@ -127,4 +129,13 @@ export class FolderUseCases {
       await this.deleteOrphansFolders(userId);
     }
   }
+
+  decryptFolderName(folder: Folder): any {
```

Comment:
Do we have this as well for files?

As in file.usecase.ts should have a decryptName

Reply:
Yes,should have it too. Adding it

Reply:
Added!